frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/donut-chart.tsx

```diff
@@ -40,7 +41,14 @@ export const DiscoveryDonutChart: React.FC<DiscoveryDonutChartProps> = ({ state,
         if (nodes.includes(disk.node)) {
           const isValidSize =
             Number(disk.size) >= minSize && (maxSize ? Number(disk.size) <= maxSize : true);
-          if (isValidSize) {
+
+            // For disk type All it always true
+            let diskTypeMatched: boolean = true;
+            if (DISK_TYPES[state.diskType]?.property) {
+              diskTypeMatched = DISK_TYPES[state.diskType].property === disk.property ? true : false;
```

Comment:
```suggestion
              hasDiskType = DISK_TYPES[state.diskType].property === disk.property;
```

Reply:
ack

Reply:
My bad, I dont know why i put true and false here :)